proto/vtctldata.proto

```diff
@@ -459,6 +458,15 @@ message CompleteSchemaMigrationResponse {
   map<string, uint64> rows_affected_by_shard = 1;
 }
 
+message ForceCutOverSchemaMigrationRequest {
```

Comment:
nit: move these down below `FindAllShardsInKeyspaceRequest/Response` (to keep them tidy)?